sunpy/time/tests/test_time.py

```diff
@@ -145,45 +186,140 @@ def test_parse_time_now():
     """
     # TODO: once mocking support is merged in, we can perform a like for like comparison,
```

Comment:
is this comment still relevant?

Reply:
I'm not sure what is meant by `mocking support`.

Reply:
me either in this context.

Reply:
I can't think of a way to check equality. If it helps, that comment was added in https://github.com/sunpy/sunpy/commit/9ffa99f154a1878dd219b8352ab2a591691fec53.

Reply:
:woman_shrugging: I think we should just remove it.